src/common/Widget.js

```diff
@@ -3,6 +3,20 @@
     if (typeof define === "function" && define.amd) {
         define(["d3/d3"], factory);
     } else {
+        root.require = root.require || function (paths, cb) {
```

Comment:
This shim doesn't handle word-cloud case yet. Do you want it fixed in this pull request or expect another pull request?